source/features/enable-file-links-in-compare-view.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import GitBranchIcon from 'octicon/git-branch.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import GitHubURL from '../github-helpers/github-url';
+
+function handlePRMenuOpening(event: delegate.Event): void {
+	event.delegateTarget.classList.add('rgh-actionable-link'); // Mark this as processed
+
+	// This solution accounts for:
+	// - Branches with slashes in it
+	// - PRs opened from the default branch
+	const dropdown = event.delegateTarget.nextElementSibling!;
+	const viewFile = select<HTMLAnchorElement>('[data-ga-click^="View file"]', dropdown)!;
+	const headBranchUrl = select<HTMLAnchorElement>('.commit-ref.head-ref a')!.pathname;
```

Comment:
```suggestion
	const headBranchUrl = select<HTMLLinkElement>('[type="application/atom+xml"]')!.href.replace(/\.atom$/, '');
```

Should fix https://github.com/sindresorhus/refined-github/issues/3123

Side note: it looks like `getCurrentBranch` works on PRs as well.

Reply:
I made the same mistake before https://github.com/sindresorhus/refined-github/pull/3111, that gives you the local repo url  `sindresorhus/refined-github/` vs `yakov116/refined-github/`

Reply:
Ugh PRs are awkward. GitHub treats your branch as a branch of this repo, which is what threw me off

```
https://github.com/sindresorhus/refined-github/commits/enable-file-links-in-compare-view.atom
```

Therefore, for PRs between default branches, the result of `getCurrentBranch` is completely wrong. Perhaps `getCurrentBranch` _cannot_ be used in PRs.

Example: in https://github.com/yuzu-emu/yuzu/pull/3953/files

```
https://github.com/yuzu-emu/yuzu/commits/master.atom
```
should be

```
https://github.com/GodKratos/yuzu/commits/master.atom
```

Reply:
Which is what let me to [this](https://github.com/sindresorhus/refined-github/pull/3152/commits/650bde6e8356e89d57b6778edb782f3e05ce4272)

>Therefore, for PRs between default branches, the result of getCurrentBranch is completely wrong. Perhaps getCurrentBranch cannot be used in PRs.

The branch is correct but the headrepo is not